From 6f88a6cc47b5a8cc245c5bcc63ca56591687aa52 Mon Sep 17 00:00:00 2001
From: Srinivas Neeli <srinivas.neeli@xilinx.com>
Date: Fri, 31 Jul 2020 11:57:52 +0530
Subject: [PATCH 1485/1851] can: xilinx_can: Add comment for spinlock_t

commit e9a6d58d2c259d42ffe6d324ad0ddad0fe09c0d1 from
https://github.com/Xilinx/linux-xlnx.git

Add comment for spinlock_t, to address checkpatch check.

Signed-off-by: Srinivas Neeli <srinivas.neeli@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/net/can/xilinx_can.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/can/xilinx_can.c b/drivers/net/can/xilinx_can.c
index e8e712d9de14..5586f6079f53 100644
--- a/drivers/net/can/xilinx_can.c
+++ b/drivers/net/can/xilinx_can.c
@@ -195,7 +195,7 @@ struct xcan_devtype_data {
  */
 struct xcan_priv {
 	struct can_priv can;
-	spinlock_t tx_lock;
+	spinlock_t tx_lock;	/* To protect tx path */
 	unsigned int tx_head;
 	unsigned int tx_tail;
 	unsigned int tx_max;
-- 
2.31.1

